iOS Client/DevotionsController/MBDevotionsViewController.swift

```diff
@@ -49,12 +77,36 @@ class MBDevotionsViewController: UIViewController, StoreSubscriber, UITableViewD
         case .loading:
             break
         case .loaded(let loadedDevotions):
-            self.devotions = loadedDevotions
-            tableView.reloadData()
+            if self.devotions.count != loadedDevotions.count {
+                self.devotions = loadedDevotions
+                tableView.reloadData()
+            } else {
+                for (idx, loadedDevotion) in loadedDevotions.enumerated() {
+                    if self.devotions[idx] != loadedDevotion {
+                        self.devotions[idx] = loadedDevotion
+                        tableView.reloadRows(at: [IndexPath(row: idx, section: 0)], with: .automatic)
+                    }
```

Comment:
should we batch up the indexes that need to be reloaded and then reload outside of the `for` loop? Probably a tad more efficient than calling `reloadRows` for each devotion that has changed.

Reply:
ooooooo @jdwitten strong. I'll address this.